languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -35785,6 +35785,41 @@ though, and has since been slightly modified)-->
                 <suggestion>c'est</suggestion>
                 <example correction="C'est"><marker>Cette</marker> ma carte.</example>
 			</rule>
+			<rule default="temp_off">
+                <pattern>
+                    <token regexp="yes">l[ae]</token>
+                    <token regexp="yes" case_sensitive="yes">[cdlmst]es|[nv]os|cette|ce[st]|[mts]a|mon|je|tu|ils?|elle?|[vn]ous|on</token>
+                </pattern>
+                <message>Les mots « \1 \2 » ne peuvent pas se suivre. L'un des deux peut donc être supprimé.</message>
+                <suggestion>\1</suggestion>
+                <suggestion>\2</suggestion>
+                <example correction="la|cette">Pour tout dire, <marker>la cette</marker> langue va disparaitre.</example>
+                <example>Plusieurs de ces langues vont disparaitre.</example>
+			</rule>
+			<rule default="temp_off">
+            <antipattern>
+                <token regexp="yes">[cdlmst]es|[nv]os|cette|ce[st]|[mts]a|[tsm]on|leur</token>
+                <token regexp="yes">(?-i)[A-Z].*</token>
+            </antipattern>
+            <antipattern>
+                <token postag="[DP].*" postag_regexp="yes"/>
+                <token regexp="yes">[st]on</token>
+                <token postag="V ppa m sp?" postag_regexp="yes"/>
+            </antipattern>
+            <pattern>
+                <token regexp="yes" case_sensitive="yes">[cdlmst]es|[nv]os|cette|ce[st]|[ts]a|ton</token>
+                <marker>
+                    <token postag="V ppa m sp?" postag_regexp="yes"><exception postag="N.*|V (ind|sub|con).*" postag_regexp="yes"/></token>
+                </marker>
+            </pattern>
+            <filter class="org.languagetool.rules.fr.FindSuggestionsFilter" args="wordFrom:2 desiredPostag:N.*"/>
```

Comment:
It would be perfect if we could tell the `FindSuggestionsFilter` that we're only looking for words that match `\1` in number and gender…